src/common/mangataHelper.js

```diff
@@ -1,13 +1,14 @@
 import _ from 'lodash';
 import BN from 'bn.js';
-import { Mangata } from '@mangata-finance/sdk';
 import { getDecimalBN, sendExtrinsic } from './utils';
 import ChainHelper from './chainHelper';
+import { dynamicImport } from './esm-dynamic-imports';
 
 class MangataHelper extends ChainHelper {
     initialize = async () => {
-        this.mangataSdk = Mangata.getInstance([this.config.endpoint]);
-        this.api = await this.mangataSdk.getApi();
+        const { Mangata } = await dynamicImport('@mangata-finance/sdk');
```

Comment:
static import at line 1 like below doesn’t work?

```
import { Managata } from '@mangata-finance/sdk';
```

Reply:
`@mangata-finance/sdk` is an [ESM-only](https://github.com/gasp-xyz/mangata-dev-kit?tab=readme-ov-file#basic-use-case) module, so it can only be imported via dynamic import in CommonJS projects (which is the default for Node.js).